targets/TARGET_STM/stm_spi_api.c

```diff
@@ -602,6 +606,25 @@ static const uint32_t baudrate_prescaler_table[] =  {SPI_BAUDRATEPRESCALER_2,
                                                      SPI_BAUDRATEPRESCALER_256
                                                     };
 
+static uint8_t spi_get_baudrate_prescaler_rank(uint32_t value)
+{
+#if TARGET_STM32H7
+    return value >> 28 & 0x07;
```

Comment:
I'd prefer to see HAL defines rather than hard coded values.

Reply:
Replaced with switch...case statement.